stats/measure.go

```diff
@@ -24,8 +24,12 @@ package stats
 // NewMeasureFloat64 automatically registers the measure
 // by the given name.
 // Each registered measure needs to be unique by name.
+// Measures also have units that represent the unit of the
+// metric they are representing.
 type Measure interface {
 	Name() string
+	Description() string
+	Unit() string
```

Comment:
Oh, Unit was already defined as a public function but not added in the interface.

Reply:
Yes.

Reply:
To be more accurate, unit was already in the concrete types but not in the interface.